cli/flox/doc/manifest.toml.md

```diff
@@ -0,0 +1,310 @@
+otherwise installation will fail.
+The most common option will likely be the `semver` option,
+which allows you to specify a semantic version range.
+
+Each option is described below:
+
+`name`
+:   Matches either the last attribute of the `pkg-path` or the package metadata
+    fields `name` or `pname` as set by the catalog.
+    This option is mutually exclusive with the `pkg-path` and `abs-path`
+    options.
+    You shouldn't need to use this option and should instead prefer the
+    `pkg-path` option.
+
+`optional`
+:   Marks this package as only an optional requirement for the environment.
+    By default an environment will fail to build if a specified package can't
+    be found in the catalog.
+    However, some packages are platform specific and will never be found in the
+    catalog for some systems.
+    Thus, the way you mark a package as platform specific is by setting
+    `opitonal = true` and using the `systems` option to list the systems on
+    which the package is required.
+
+`pkg-group`
+:   Marks a package as belonging to a group of packages.
+    Some packages only work together correctly when they all come from the
+    same revision of the catalog.
+    One example is C/C++ projects that depend on specific versions of header
+    files.
+    Packages are marked as belonging to a group simply by setting this option
+    to the name of the group.
+
+    Packages that are marked as belonging to the same group will always be
+    installed from the same revision of the catalog.
+    Multiple package groups may resolve to the same catalog revision.
+    Package groups may be upgraded as a unit,
+    ensuring that the group continues to work together.
+    See [`flox-upgrade(1)`](./flox-upgrade.md) for more details on how groups
+    and packages interact during upgrades.
+
+`version`
+:   Requires that the package match either an exact version or a semver range.
+    When the first character of the `version` string is '=' the version must be
+    an exact match,
+    otherwise the `version` string is matched as a semver range.
+    Versions that don't conform to semver must be specified with '='.
+    
+    The semantic version can be specified with the typical qualifiers such as
+    `^`, `>=`, etc.
+    Semantic versions that do not specify all three fields
+    (`MAJOR.MINOR.PATCH`) will treat the unspecified fields as wildcards.
+    This instructs Flox to find the latest versions for those fields.
+    For example `version = "1.2"` would select the latest version in the
+    `1.2.X` series.
+
+    This option is mutually exclusive with the `semver` option.
+
+`semver`
+:   This option is similar to `version` except it _only_ allows semantic
+    versions.
+
+`systems`
+:   A list of systems on which to install this package.
+    When omitted this defaults to the same systems that the environment
+    specifies that it supports.
+
+`pkg-path`
+:   The abbreviated location of a package within a catalog.
```

Comment:
abbreviated from what?
maybe describe this _after_ `abs-path`. Then the description can be shortened to `abs-path` but with the first two attributes inferred